content/library_constants.tex

```diff
@@ -39,6 +39,15 @@
 See Section~\ref{subsec:thread_support} for more detail about its use.
 \tabularnewline \hline
 %%
+\LibConstDecl{\newtext{SHMEM\_CTX\_INVALID}} &
+\newtext{
+A value to which a context handle may be initialized, assigned, or
```

Comment:
The style here doesn't match the other table entries, which start with a sentence fragment.

I'm also concerned about the last part, "test whether it is *an* invalid communication context".  `SHMEM_CTX_INVALID` is one of many invalid handle values, but the statement suggests that it subsumes all invalid handle values.  I wonder whether something like `SHMEM_CTX_NULL` would be less prone to misinterpretation, since what this text describes is similar to how the null pointer value is used?

Here's a suggested update:
```
A value corresponding to an invalid communication context.  This value can be used to initialize or update context handles to indicate that they do not reference a valid context.  When managed in this way, applications can use an equality comparison to test whether a given context handle references a valid context.
```